src/api/expense.controller.js

```diff
@@ -0,0 +1,92 @@
+const { expenseService } = require('../services/expense.service.js');
+const { userService } = require('../services/user.service.js');
+
+const getExpenses = async (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const categoryList = categories ? categories.split(',') : [];
+
+  const expenses = await expenseService.getExpenses({
+    userId,
+    categoryList,
+    from,
+    to,
+  });
+
+  res.json(expenses);
+};
+
+const createExpense = async (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+
+  if (!userId || !spentAt || !title || !amount) {
+    return res.sendStatus(400);
+  }
+
+  const user = await userService.getUserById(userId);
+
+  if (!user) {
+    return res.sendStatus(400);
```

Comment:
Similar to the previous comment, consider sending a JSON response with an error message to indicate that the user was not found.